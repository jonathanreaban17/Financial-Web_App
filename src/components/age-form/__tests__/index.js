import React from "react";
import { AgeForm } from "../";
import renderer from "react-test-renderer";

it("renders correctly", () => {
  const tree = renderer
    .create(<AgeForm lifeExpectancy={90} startingAge={28} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
