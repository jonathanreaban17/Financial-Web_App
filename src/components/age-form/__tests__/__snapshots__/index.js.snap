// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-bdVaJa cWaPdO"
>
  <span>
    I am
  </span>
  <input
    className="sc-bwzfXH hOVJAs"
    type="number"
    value={28}
  />
  <span>
    years old with a life expectancy of
  </span>
  <input
    className="sc-bwzfXH hOVJAs"
    type="number"
    value={90}
  />
</div>
`;
